server/events/project_finder.go

```diff
@@ -48,12 +47,16 @@ var ignoredFilenameFragments = []string{"terraform.tfstate", "terraform.tfstate.
 type DefaultProjectFinder struct{}
 
 // See ProjectFinder.DetermineProjects.
-func (p *DefaultProjectFinder) DetermineProjects(log *logging.SimpleLogger, modifiedFiles []string, repoFullName string, absRepoDir string) []models.Project {
+func (p *DefaultProjectFinder) DetermineProjects(log *logging.SimpleLogger, modifiedFiles []string, repoFullName string, absRepoDir string, autoplanFileList string) ([]models.Project, error) {
 	var projects []models.Project
 
-	modifiedTerraformFiles := p.filterToTerraform(modifiedFiles)
+	modifiedTerraformFiles, err := p.filterToFileList(log, modifiedFiles, autoplanFileList)
+	if err != nil {
+		return nil, errors.Wrapf(err, "filtering modified files by autoplanFileList: %s", autoplanFileList)
```

Comment:
seems unnecessary to wrap this error again, how about logging it instead?

Reply:
I wasn't sure what to do here. Contributing docs suggested always wrapping errors. When I move the default value out of the function. I'm going to add the syntax error checking to the attribute validation step. That would mean we don't need to worry about errors here anymore, so this will go away.